### PR TITLE
Use mirror copy of SDF data

### PR DESCRIPTION
**Pull request by Xarthisius**
Trying to speed up SDF test by using NCSA mirror.

### CODE REVIEW

**Comment by matthewturk** [comment]
LGTM

File: yt/frontends/sdf/tests/test_outputs.py [hunk 1]
```diff
@@ -40,7 +41,7 @@ def internet_on():
 def test_scivis():
     if not internet_on():
```
Comment (munkm) [inline yt/frontends/sdf/tests/test_outputs.py]: the internet_on function has a call to the `scivis_data`. If this PR switches to the mirror should that function be updated to the mirror?

Comment (Xarthisius) [reply yt/frontends/sdf/tests/test_outputs.py]: good catch! Thanks!

**Comment by munkm** [comment]
It looks like the line with `assert_equal(str(ds), "ds14_scivis_0128_e4_dt04_1.0000")` is causing a failure because the string of the new ds is associated with the yt.upload name `744abba3`.

**Comment by matthewturk** [comment]
Since we're not having a lot of success with getting this to work with the mirror, I am going to push to this PR a change that disables the test.

**Pull request merged**